### PR TITLE
Change message when not subscribe to notification (#361)

### LINKED ISSUE #361: Test-App could use a friendly message when trying to remove a subscription that does not exist
For the Test-App, When running the command to remove a subscription that does not exist it would be nice to have a friendlier message. 

The current `not found` is confusing as it looks like an error has occurred.

image

### PR DESCRIPTION
**Pull request by javaguirre**
#### Summary

I was learning about the test app, fixes #361

### CODE REVIEW

File: server/appservices/subscriptions.go [hunk 1]
```diff
@@ -166,7 +166,7 @@ func (a *AppServices) unsubscribe(r *incoming.Request, ownerUserID string, e app
 		return modified, nil
 	}
 
-	return all, utils.ErrNotFound
+	return all, errors.New("You are not subscribed to this notification/s")
```
Comment (hanzei) [inline server/appservices/subscriptions.go:169]: Why the `/s`? Does the command accept more than one subscription?

Comment (hanzei) [inline server/appservices/subscriptions.go:169]: 2/5 that we should still `errors.Wrap` `utils.ErrNotFound` so the upper layers can check for that error.

**Review by hanzei — CHANGES_REQUESTED** [review]

**Comment by hanzei** [comment]
And a test needs to be fixed

File: server/appservices/subscriptions.go (hunk 1 above)
Comment (javaguirre) [reply server/appservices/subscriptions.go:169]: mmm, I don't understand what you mean, but I'll change it.

**Review by hanzei — APPROVED** [review]

File: server/appservices/subscriptions.go (hunk 1 above)
Comment (levb) [reply server/appservices/subscriptions.go]: @javaguirre see `errors.Cause`

**Review by levb — APPROVED** [review]

**Pull request merged**